install/kubernetes/cilium/charts/hubble-relay/values.yaml

```diff
@@ -1,5 +1,8 @@
 image: hubble-relay
 
+# Enables the hubble-relay Deployment
+enabled: false
```

Comment:
nits: tag, pullpolicy are still refering to global values. I can send PR later if you are ok.